### ISSUE TITLE
ci: failed tests due to FileNotFoundError

### ISSUE THREAD

**Issue by wang-boyu** [ABSESpy/ABSESpy#45]
Test cases are part of the [joss review process](https://github.com/openjournals/joss-reviews/issues/6298). Overall the package has a pretty good test coverage of 86%. Two tests are failing due to missing data:

```bash
=================================== short test summary info ===================================
ERROR tests/api/test_dynamic.py::TestDynamicData::test_dynamic_city_lands - FileNotFoundError: [Errno 2] No such file or directory: 'data/water_quota/irr_lands.csv'
ERROR tests/api/test_dynamic.py::TestDynamicData::test_dynamic_nc_data - FileNotFoundError: [Errno 2] No such file or directory: 'data/water_quota/irr_lands.csv'
```

I would also recommend including these tests in GitHub actions (which currently is configured to build and deploy GitHub pages), so that tests can be run automatically. But this is absolutely not required and up to the authors.

**Comment by SongshGeo** [comment]
👋 Hi, Boyu, Thanks for reporting this issue.
I will refactor the tests with smaller pieces of data requirements.

**Comment by SongshGeo** [comment]
Should be fixed in #78
